flatpak-repos/Makefile.am

```diff
@@ -0,0 +1,7 @@
+flatpakrepodir = $(datarootdir)/eos-flatpak-repos
```

Comment:
I would recommend using `$(pkgdatadir)` for this

Reply:
Alright. I kind of hate that this is all in eos-boot-helper but we can move them around whenever we want as this is an exclusively ostree-controlled path anyway.